integrations/clickup/client.py

```diff
@@ -0,0 +1,190 @@
+from typing import Any, AsyncGenerator, Dict, List
+
+from httpx import Timeout
+
+from loguru import logger
+
+
+from port_ocean.utils import http_async_client
+
+
+class ClickUpClient:
+    def __init__(self, clickup_token: str) -> None:
+        self.clickup_url = "https://api.clickup.com"
```

Comment:
Why are you storing this as a class variable? You can use it directly. Also, I prefer you load it from the integration config, just like you are doing for the clickup_token